src/cloud-api-adaptor/test/e2e/common.go

```diff
@@ -89,13 +89,25 @@ default WriteStreamRequest := true
 
 func buildInitdataBody(kbsEndpoint string) (string, error) {
 	content, err := os.ReadFile("../trustee/kbs/config/kubernetes/base/https-cert.pem")
+
 	if err != nil {
 		return "", err
 	}
 	certContent := string(content)
 	body := fmt.Sprintf(testInitdata, kbsEndpoint, kbsEndpoint, certContent, kbsEndpoint, certContent)
 	return body, nil
 }
+func buildInitdataBodyFromEmptyInitdata(kbsEndpoint string) (string, error) {
```

Comment:
This isn't what I suggested - you have too much duplicated code here and have hardcoded the initdata string?